bin/alluxio-mount.sh

```diff
@@ -117,7 +107,7 @@ function mount_ramfs_mac() {
 
 function umount_ramfs_mac() {
   TIER_PATH=${1}
-  local device=$(df -l | grep ${TIER_PATH} | cut -d " " -f 1)
+  local device=$(df -l | grep -w ${TIER_PATH} | cut -d " " -f 1)
```

Comment:
why add `-w` to the greps?

Reply:
because a normal grep can return multiple entries.. when you grep for `Volumes/ramdisk` it returns both `/Volumes/ramdisk` and `/Volumes/ramdisk2`, and it confuses the subsequent script

Reply:
I think my main concern is that the `-w` is not an option specified for `grep` in the posix standard - so it makes the script less portable. I am hesitant about using this.

Do you think there is another way that has higher portability?

Reply:
This is the latest spec for `grep`: https://pubs.opengroup.org/onlinepubs/9699919799/

Reply:
changed to using `-E "(^|[^A-Za-z0-9_])${TIER_PATH}($|[^A-Za-z0-9_])"`

Reply:
what about `grep -E "(^|[[:space:]])${TIER_PATH}($|[[:space:]])"`